### ISSUE TITLE
Brackets indent guide appears with dashed lines instead of straight

### ISSUE THREAD

**Issue by frankny** [lkcampbell/brackets-indent-guides#36]
Any ideas how to fix this? Image attached displaying behavior. Thanks
![e25039ce-49d8-11e7-87a2-b4cddb252ec5](https://cloud.githubusercontent.com/assets/27034893/26824653/3fdd2f32-4a80-11e7-959a-2e01054f2c0f.png)

**Comment by lkcampbell** [comment]
@frankny, I know @tweakimp has been looking at ways to dynamically size the guide segments for issue #34 but I haven't investigated his solution yet. Maybe he can tell us if it is applicable to this problem as well.

The only workaround I know of right now is to use different fonts or themes. The gaps are being caused by the extra space between the lines, probably a style of the theme.

**Comment by tweakimp** [comment]
Please tell us the theme and the settings `fonts.fontSize` and `fonts.fontFamily` you are using.
Maybe a theme could widen the space between lines while the font stays the same so that these gaps are created.

**Comment by frankny** [comment]
@lkcampbell @tweakimp   Yes, depending on what themes I use the problem does go away. I'm using 'smart theme dark' and my font is 'Andale Mono'. I'm kind of a stickler for editor appearance and spent a decent amount of time getting it to look how I want, so changing the appearance so that the extension works wouldn't be an option for me.

**Comment by tweakimp** [comment]
In `smart theme dark` settings file `theme.less` you can set `
.CodeMirror,
.CodeMirror-scroll {
	line-height: 15px;
}
` which will fix the problem but obviously make each line height smaller.

**Comment by frankny** [comment]
@tweakimp thanks ill try it out.

**Comment by lkcampbell** [comment]
@tweakimp, thanks for your input.

@frankny, the Indent Guides extension is built on CodeMirror Overlay API. Unfortunately, I am limited by that API. It has constantly introduced problems with the overlap or spread of the indent guide segments, something I have struggled with for years.

Your request is reasonable, but it is beyond my ability to solve, so I am am going to close this bug as a Code Challenge. Maybe someday I will convince the creator of CodeMirror to create Native Indent Guides, but until that happens, this is the hack job you get from me :).

**Comment by frankny** [comment]
@lkcampbell no worries, thanks anyways.

**Comment by lkcampbell** [comment]
I have a couple of ideas on how to solve this problem, so I am reopening the issue to investigate.

**Comment by netoper** [comment]
problem solved ;)
no matter on the font size or line height.. i use line-height: 1.5px; on screenshot.

![lk](https://user-images.githubusercontent.com/32483343/33073104-0d89b368-cec2-11e7-954e-fa53218de00e.jpg)

thank you, you made my day :)

**Comment by Michael-Luxus** [comment]
waw, thank you!

**Comment by lkcampbell** [comment]
@netoper I know I used that border solution some time in the past and ran into problems. Unfortunately, it was so long ago that I can't remember what the problems were. At any rate, I will be glad to take a look at it again and see if it will solve the problem now.
